lib/import/brca/providers/liverpool/liverpool_handler.rb

```diff
@@ -0,0 +1,92 @@
+require 'possibly'
+require 'pry'
+
+module Import
+  module Brca
+    module Providers
+      module Liverpool
+        # Process Liverpool-specific record details into generalized internal genotype format
+        class LiverpoolHandler < Import::Brca::Core::ProviderHandler
+          include Import::Helpers::Brca::Providers::Rep::Constants
+
+          def process_fields(record)
+            # return for colorectal cases
+            return if record.raw_fields['investigation'].match(/HNPCC|Peutz-Jegher\sSyndrome/i)
+
+            genotype = Import::Brca::Core::GenotypeBrca.new(record)
+            genotype.add_passthrough_fields(record.mapped_fields,
+                                            record.raw_fields,
+                                            PASS_THROUGH_FIELDS)
+            add_organisationcode_testresult(genotype)
+            add_genetictestscope(genotype, record)
+            add_test_status(genotype, record)
+            result = process_variants_from_record(genotype, record)
+            result.each { |cur_genotype| @persister.integrate_and_store(cur_genotype) }
+          end
+
+          def add_organisationcode_testresult(genotype)
```

Comment:
@shilpigoeldev I'm not sure this needs to be it's own method. I think it's only used once, and is a single line.